test_shard/fake_test_app/integration_test/generated/mediaplayer_audiopcmframesink_testcases.generated.dart

```diff
@@ -62,7 +63,20 @@ void generatedTestCases(IrisTester irisTester) {
           'frame': frame.toJson(),
         };
 
-        irisTester.fireEvent('AudioPcmFrameSink_onFrame', params: eventJson);
+        if (!kIsWeb) {
```

Comment:
Throughout all these updates there is a lot of (!kIsWeb), is the case that `kIsWeb == true` is all that handled?